src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -232,9 +233,14 @@ public Vector2 getGyroscopeXY() {
         return Vector2.ZERO;
     }
 
-    public Rotation2 getGyroscopeUnadjustedAngle() {
+    /**
+     * Returns the unadjusted gyroscope angle.
+     *
+     * @return Unadjusted gyroscope angle (yaw, positive rotation is clockwise)
+     */
+    public Rotation2d getGyroscopeUnadjustedAngle() {
         synchronized (sensorLock) {
-            return gyroscope.getUnadjustedAngle();
+            return Rotation2d.fromDegrees(gyroscope.getUnadjustedAngle().toDegrees());
```

Comment:
Could we use the constructor with radians instead of having degrees appear twice?

Reply:
Yes, but I thought that using `fromDegrees()` and `toDegrees()` would be clearer.